site2/docs/administration-metadata-store.md

```diff
@@ -0,0 +1,120 @@
+---
+id: administration-metadata-store
+title: Configure metadata store
+sidebar_label: "Configure metadata store"
+---
+
+Pulsar metadata store maintains all the metadata, configuration and coordination of a Pulsar cluster, such as topic metadata, schema, broker load data, and so on. 
```

Comment:
```suggestion
Pulsar metadata store maintains all the metadata, configuration, and coordination of a Pulsar cluster, such as topic metadata, schema, broker load data, and so on. 
```
Use commas to separate items in a series of three or more. Use a comma before the conjunction that precedes the final item.
https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.b82f2ay5cpsc

Reply:
Done. Thank you.